pkg/controller/plan/validation.go

```diff
@@ -573,6 +575,21 @@ func (r *Reconciler) validateVM(plan *api.Plan) error {
 		Status:   True,
 		Category: Warn,
 		Message:  "VMs with shared disk can not be migrated.", // This should be set by the provider validator
+	}
+	unorderedNics := libcnd.Condition{
+		Type:     NetMapOrder,
+		Status:   True,
+		Reason:   NotValid,
+		Category: Warn,
+		Message:  "VM network mapping is unordered.",
```

Comment:
`VM network mapping is unordered, this can cause the interface names to be different after migration.`